<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelWidget</class>
 <widget class="QWidget" name="PanelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Chart</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="9" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>General:</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Theme:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="themeComboBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Type:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="seriesType">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>Line</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Spline</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Scatter</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Legend position:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="legendPosition">
            <item>
             <property name="text">
              <string>Top</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Right</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bottom</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Left</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>View:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="animation">
            <property name="text">
             <string>Animation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="antialiasing">
            <property name="text">
             <string>Antialiasing</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Tick count:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="1">
           <widget class="QSlider" name="tickCountY">
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="tickCountX">
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="spinBoxTickX">
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="showGroupSeparator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="spinBoxTickY">
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Rubber band:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QRadioButton" name="rubberModeRectangle">
            <property name="text">
             <string>Rectangle</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rubberModeHorizontal">
            <property name="text">
             <string>Horizontal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rubberModeVertical">
            <property name="text">
             <string>Vertical</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Title:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="title">
            <property name="text">
             <string/>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Axises range:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="axisXRangeMin">
            <property name="minimum">
             <double>-100000000000000004764729344.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000000000000000044885712678075916785549312.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="axisXRangeMax">
            <property name="minimum">
             <double>-1000000000000000013287555072.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9999999999999999635896294965248.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="axisYRangeMin">
            <property name="minimum">
             <double>-99999999999999991433150857216.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000000000000000019884624838656.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="axisYRangeMax">
            <property name="minimum">
             <double>-100000000000000004764729344.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000000000000000019884624838656.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_series">
      <attribute name="title">
       <string>Series</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="1">
        <widget class="QPushButton" name="deleteSeries">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Delete series</string>
         </property>
         <property name="icon">
          <iconset resource="../img.qrc">
           <normaloff>:/trash</normaloff>:/trash</iconset>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Points:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="1">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="saveToCSV">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Save to CSV</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="4">
           <widget class="QTableView" name="tableView"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Width:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QSlider" name="penWidth">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>48</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxPenWidth">
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>48</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="propertyBox">
         <property name="title">
          <string>Property:</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="pointVisible">
            <property name="text">
             <string>point visible</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="colorSelect">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Color:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="hide">
            <property name="text">
             <string>hide</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QComboBox" name="seriesList"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionSetRubberMode">
   <property name="text">
    <string>SetRubberMode</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>themeComboBox</tabstop>
  <tabstop>seriesType</tabstop>
  <tabstop>legendPosition</tabstop>
  <tabstop>title</tabstop>
  <tabstop>animation</tabstop>
  <tabstop>antialiasing</tabstop>
  <tabstop>tickCountX</tabstop>
  <tabstop>tickCountY</tabstop>
  <tabstop>seriesList</tabstop>
  <tabstop>colorSelect</tabstop>
  <tabstop>pointVisible</tabstop>
  <tabstop>hide</tabstop>
  <tabstop>spinBoxTickX</tabstop>
  <tabstop>spinBoxTickY</tabstop>
 </tabstops>
 <resources>
  <include location="../img.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tickCountX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxTickX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tickCountY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxTickY</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>penWidth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxPenWidth</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxPenWidth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>penWidth</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxTickX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>tickCountX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxTickY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>tickCountY</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rubberModeVertical</sender>
   <signal>clicked()</signal>
   <receiver>PanelWidget</receiver>
   <slot>on_actionSetRubberMode_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rubberModeHorizontal</sender>
   <signal>clicked()</signal>
   <receiver>PanelWidget</receiver>
   <slot>on_actionSetRubberMode_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rubberModeRectangle</sender>
   <signal>clicked()</signal>
   <receiver>PanelWidget</receiver>
   <slot>on_actionSetRubberMode_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>axisXRangeMin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PanelWidget</receiver>
   <slot>slotAxisXRangeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>538</x>
     <y>515</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>axisXRangeMax</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PanelWidget</receiver>
   <slot>slotAxisXRangeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>axisYRangeMin</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PanelWidget</receiver>
   <slot>slotAxisYRangeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>axisYRangeMax</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PanelWidget</receiver>
   <slot>slotAxisYRangeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signalChangeTheme(QChart::ChartTheme)</signal>
  <slot>on_actionSetRubberMode_triggered()</slot>
  <slot>slotAxisXRangeChanged()</slot>
  <slot>slotAxisYRangeChanged()</slot>
 </slots>
</ui>
